frontend/server/src/Controllers/Badge.php

```diff
@@ -145,6 +146,25 @@ private static function getBadgeDetails(string $badgeAlias): array {
             'owners_count' => $ownersCount,
         ];
     }
+    /**
+     * @return array{smartyProperties: array{payload: BadgeListPayload, title: string}, entrypoint: string}
+     */
+    public static function getBadgeListForSmarty(\OmegaUp\Request $r) {
+        $r->ensureIdentity();
+        $badges = self::apiList($r);
+        $owned_badges = self::apiMyList($r);
```

Comment:
nit:

```suggestion
        $ownedBadges = self::apiMyList($r);
```

en PHP, todas las variables deben ir en camelCase.